src/index.rs

```diff
@@ -434,43 +435,41 @@ impl Index {
     /// into the current one
     pub fn import(&mut self, mut other: Index) {
         //global variables
-        for (name, mut e) in other.global_variables.drain(..) {
-            e.initial_value =
-                self.maybe_import_const_expr(&mut other.constant_expressions, &e.initial_value);
+        for (name, e) in other.global_variables.drain(..) {
+            let e = self.transfer_constants(e, &mut other.constant_expressions);
             self.global_variables.insert(name, e);
         }
 
-        //enum_global_variables
-        for (name, mut e) in other.enum_global_variables.drain(..) {
-            e.initial_value =
-                self.maybe_import_const_expr(&mut other.constant_expressions, &e.initial_value);
-            self.enum_global_variables.insert(name, e.clone());
-            self.enum_qualified_variables
-                .insert(e.qualified_name.to_lowercase(), e);
+        //enmu_variables use the qualified variables since name conflicts will be overriden in the enum_global
+        for (qualified_name, e) in other.enum_qualified_variables.drain(..) {
+            let e = self.transfer_constants(e, &mut other.constant_expressions);
+            dbg!(&e);
```

Comment:
remove dbg!
btw. is this code for another issue? or is it related to builtins?

Reply:
Since we agreed to not have the builtins sneaked into the index when it's created but have them be parsed independently the import issue has to be solved for the builtins to work.
I could still split the issue in 2 but the import has to come first